src/aiida/cmdline/utils/common.py

```diff
@@ -486,13 +486,17 @@ def build_entries(ports):
         echo.echo(style('\nExit codes that invalidate the cache are marked in bold red.\n', italic=True))
 
 
-def generate_validate_output_file(
-    output_file: Path | None, entity_label: str, appendix: str = '', overwrite: bool = False
+def validate_output_filename(
+    output_file: Path | str | None,
```

Comment:
If you don't accept `None`, I wouldn't add it in the type hint

Reply:
Good catch! Fixed now.